filebeat/prospector/prospector.go

```diff
@@ -77,7 +82,7 @@ func (p *Prospector) initProspectorer(outlet channel.Outleter, states []file.Sta
 	case harvester.RedisType:
 		prospectorer, err = redis.NewProspector(config, outlet)
 	case harvester.LogType:
-		prospectorer, err = log.NewProspector(config, states, outlet, p.done)
+		prospectorer, err = log.NewProspector(config, states, outlet, p.done, p.beatDone)
```

Comment:
In the past the p.beatDone was already part of the outlet. I assume this changes now as the prospector itselfs creates different outlets?